conf.d/file.yaml.example

```diff
@@ -1,13 +1,15 @@
 # init_config:
-# # Not required for this check
 #
 # instances:
 #  - path: '/var/run/reboot-required'
 #    expected: absent
 #  - path: '/etc/passwd'
 #    expected: present
+#    Optionally require that the file be at least this many seconds old
+#    before we call it "present".
+#    present_minimum_age_seconds: 300
 
 init_config:
   # Not required for this check
 
-instances:
\ No newline at end of file
+instances:
```

Comment:
This was broken in the previous version too, but can you put a `[]` there so it's a valid config file?